script/Arcueid/Day 7/07_00_ARC07_3F.txt

```diff
@@ -917,7 +917,7 @@ But if I let her go, I won't be able to talk to her like this anymore.
 [sha:7738fffe126e6423ac3102ad17218db13181c086]{
 -- Page 398, Offset 17569.
 -- 彼女は、自分の死を、頭にくるぐらい恐れていない。
-This girl is so unafraid of her own death that it's enough to drive me mad.
+She is so unafraid of her own death that I'm about to pop a lid.
```

Comment:
Pop a lid feels like quite the tone mismatch to me here

Reply:
Yeah, true.
I think being a bit plain is about all we can do, then. Not that there's anything wrong with that.

>She is so unafraid of her own death that it's making me angry.

Reply:
I mean I feel we can be _slightly_ less plain? I think original was fine, or 
> She is so unafraid of her own death that it's driving me insane.

to make it a bit shorter